src/nav/attribute-dictionary.yml

```
@@ -1,2 +1,5 @@
 title: Attribute Dictionary
-path: /attribute-dictionary
+path: /attribute-dictionary/
+pages:
+  - title: Events
+    path: /attribute-dictionary/
```

```suggestion
path: /attribute-dictionary
pages:
  - title: Events
    path: /attribute-dictionary
```

All of our other paths are not using a trailing slash. While the browser will show it in the URL, we should be consistent. Our nav elements should be matching regardless, so if it isn't, we have a bug.